akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingEvents.scala

```diff
@@ -120,7 +120,21 @@ private[akka] final class ReplayingEvents[C, E, S](
             def handleEvent(event: E): Unit = {
               eventForErrorReporting = OptionVal.Some(event)
               state = state.copy(seqNr = repr.sequenceNr)
-              state = state.copy(state = setup.eventHandler(state.state, event), eventSeenInInterval = true)
+
+              setup.activeActive.foreach((aa: EventSourcedBehavior.ActiveActive) => {
```

Comment:
Perhaps do a match here instead given that it is for each event also when not using active active